test/Base_Tests/src/Network/Enso_Cloud/Enso_Cloud_Spec.enso

```diff
@@ -15,22 +14,6 @@ import project.Network.Enso_Cloud.Cloud_Tests_Setup.Cloud_Tests_Setup
    $ export ENSO_HTTP_TEST_HTTPBIN_URL=http://localhost:8080/
 spec setup:Cloud_Tests_Setup =
     setup.with_prepared_environment <|
-        Test.group "Enso Cloud Basic Utils" pending=setup.pending <|
-            Test.specify "will report Not_Logged_In if no credentials file is found" <|
-                non_existent_file = (enso_project.data / "nonexistent-file") . absolute . normalize
-                non_existent_file.exists.should_be_false
-
-                Test_Environment.unsafe_with_environment_override "ENSO_CLOUD_CREDENTIALS_FILE" non_existent_file.path <|
-                    # This test has to run before any other Cloud access, otherwise the token may already be cached.
-                    Cloud_Utils.authorization_header.should_fail_with Cloud_Utils.Not_Logged_In
-
-            Test.specify "should be able to get the cloud URL from environment" <|
-                api_url = Cloud_Utils.cloud_root_uri
-                api_url.should_equal setup.api_url.to_text
-
-            Test.specify "should be able to read the authorization token" pending=setup.mock_only_pending <|
-                Cloud_Utils.authorization_header.to_display_text.should_equal "Authorization: Bearer "+Cloud_Tests_Setup.test_token
```

Comment:
I've removed these tests, because they were checking private implementation details (and I think we are not yet skipping `private` check for tests).

IMO it was OK to remove because everything relevant is covered by Enso_User tests below which _should be run on CI_ because they only rely on the `http-test-helper` mock, which **is** available on CI.

So IMO there was not much added value in these whitebox tests.